gradle/tasks/hsql.gradle

```diff
@@ -15,12 +15,14 @@ import com.wiredforcode.gradle.spawn.*
 
 // NOTE: The startHsql task can be executed after uportal-war is built because it uses hsqldb-2.3.2.jar in classpath, otherwise it will fail
 task startHsql(type: SpawnProcessTask) {
-    description "Start hsql server"
-    def cmd = "java -cp ./uportal-war/target/uPortal/WEB-INF/lib/hsqldb-2.3.2.jar org.hsqldb.server.Server --database.0 file:./data/uPortal;hsqldb.tx=mvcc --dbname.0 uPortal --address localhost --port 8887"
+    group 'Hypersonic SQL'
+    description 'Start hsql server.'
+    def cmd = "java -cp ${projectDir}/uportal-war/target/uPortal/WEB-INF/lib/hsqldb-2.3.2.jar org.hsqldb.server.Server --database.0 file:./data/uPortal;hsqldb.tx=mvcc --dbname.0 uPortal --address localhost --port 8887"
```

Comment:
In this case, the task is meant to be run from the root project. With the coming changes that may move this around, I will change this to rootDir -- thanks for finding this one!